src/templates/vibso_classes.tsv

```diff
@@ -30,7 +27,7 @@ obo:VIBSO_0000023	point distance Y	sampling pace in the Y direction|sampling ste
\ No newline at end of file
+obo:VIBSO_0000027	number of measurements Z	number of points in the Z direction|number of layers per acquisition	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/32	A setting datum that specifies how many equally configured measurements of a sample are to be performed in the vertical direction (on the Z-axis) of a certain region of interest. Together with the focal point distance, this setting defines the region of interest depth.		owl:Class	setting datum	ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3956	10	number of measurements Z = (region of interest depth) / (point distance Z)				
+obo:VIBSO_0000028	Raman raw data	raw Raman data	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/77	A measurment datum that is obtained from a calibrated and ready Raman spectrometer and prior to be submitted to any mathematical modification. The only corrections on the data shall be the one associated with the spectrometer. The data will assign pixels readouts with nm as per calibration of the spectrometer and a.u. as intensity readout but will not be corrected in relationship of the incident monochromatic beam.	https://wiki.charisma.ideaconsult.net/wiki/Item:Q64	owl:Class	measurement datum	ORCID:0000-0002-2239-3955						
+obo:VIBSO_0000029	lens numerical aperture	numerical aperture	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/76	A setting datum that, as a dimensionless number, represents the quality which characterizes the range of angles over which an optical system can accept or emit light [dc:source wikidata:Numerical_aperture]. It gets calculated by the sine of one half of the vertex angle of the largest cone of meridional rays that can enter or leave an optical system or element, multiplied by the refractive index of the medium in which the cone is located.	https://wiki.charisma.ideaconsult.net/wiki/Item:Q385|https://en.wikipedia.org/wiki/Numerical_aperture	owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955	The light was focused onto the embedded sample using a 100× objective with a numerical aperture of 0.9, resulting in a diffraction-limited in-plane laser spot size of <1 μm.		possible equivalents in other terminologies: http://purl.obolibrary.org/obo/NCIT_C181921 & https://w3id.org/reproduceme#LensNA			
+obo:VIBSO_000030	primary Raman data		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/132	A data item that is obtained by processing raw Raman data.		owl:Class	'data item'|(is_specified_output_of some ('data transformation' and (has_specified_input some 'Raman raw data')))	ORCID:0000-0002-1595-3213|ORCID:0000-0002-2239-3955
```

Comment:
I added the `(is_specified_output_of some ('data transformation' and (has_specified_input some 'Raman raw data')))` axiom to formalize the textual definition.